app/src/main/AndroidManifest.xml

```diff
@@ -311,6 +311,11 @@
     <activity
       android:name=".app.testing.activity.TestActivity"
       android:theme="@style/OppiaThemeWithoutActionBar" />
+    <activity
+      android:name=".app.survey.SurveyActivity"
+      android:label="@string/survey_activity_title"
```

Comment:
This activity doesn't really have a title, but should probably have a content description for callback.